deb-package-builder/extensions/functions.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# common functions for building extensions
+
+E_PARAM_ERR=250
+
+download_from_pecl()
+{
+    # Download the source code, rename, extract it for debian package
+    if [ -z "$1" ]; then
+        return $E_PARAM_ERR
```

Comment:
Maybe `exit $E_PARAM_ERR` after printing an error message, if you want to catch this precondition? As it is, you're not using the return value.